config/sandruinblacklist.txt

```diff
@@ -10,3 +10,4 @@
 #_maps/RandomRuins/SandRuins/crash_bar.dmm
 #_maps/RandomRuins/SandRuins/crash_cargo.dmm
 #_maps/RandomRuins/SandRuins/crash_cult.dmm
+#_maps/RandomRuins/SandRuins/pubbysloptcrash.dmm
```

Comment:
not sure if the blacklist is even used (or even works for that matter) but you should probably remove the extra t between slop and crash